doc/classes/@GlobalScope.xml

```diff
@@ -2143,49 +2143,49 @@
 			Space key.
 		</constant>
 		<constant name="KEY_EXCLAM" value="33" enum="Key">
-			! key.
+			Exclamation mark ([code]![/code]) key.
 		</constant>
 		<constant name="KEY_QUOTEDBL" value="34" enum="Key">
-			" key.
+			Quotation mark ([code]"[/code]) key.
 		</constant>
-		<constant name="KEY_NUMBERSIGN" value="35" enum="Key">
-			# key.
+		<constant name="KEY_NUMBERSIGN" value="35" enum="Key" keywords="KEY_POUND, KEY_HASH">
```

Comment:
I don't think keywords should be fake constants. Here they could be "pound, hash" I guess, but creating new API aliases doesn't seem like a good idea to me.

Reply:
Right. Note that this is not the only case of this capitalization being done. For example
https://github.com/godotengine/godot/blob/5efd124ca10bf46df62fa2441d80589777e54a5a/doc/classes/RenderingServer.xml#L5483-L5491